full/src/test/java/apoc/systemdb/SystemDbTest.java

```diff
@@ -74,5 +109,151 @@ public void testWriteStatements() {
 
         assertEquals(2l, TestUtil.count(db, "CALL apoc.systemdb.execute('SHOW USERS')"));
     }
+    
+    @Test
+    public void testExportMetadata() {
```

Comment:
ma scusa non puoi fare dei semplici test dove testiamo che data una query cypher in input l'export ritorna la medesima query?

Reply:
Sì avevo fatto così perché ad esempio con `custom.asProcedure` esporto con con la sintassi `custom.declareProcedure` (ovvero quella non deprecata), quindi non potevo controllare direttamente l'uguaglianza. Comunque vabbè, lo cambio un po' e faccio così

Reply:
Cambiato